golioth/pytest/golioth_test.py

```diff
@@ -0,0 +1,87 @@
+import os
+import pytest
+from golioth import Client, Project, Blueprint, Tag
+
+proj_name = 'firmware_ci'
+tag_name = 'pytest'
+bp_name = 'pytest_bp'
+
+api_key = os.environ['GOLIOTH_API_KEY']
+client = Client(api_url = "https://api.golioth.io", api_key = api_key)
+ci_proj = Project(client, {'id':None, 'name':None})
+
+
+@pytest.mark.asyncio
+async def test_projects():
+    global ci_proj
```

Comment:
It's generally not a good idea to have tests rely on the successful execution of other tests. In this case, if we skip the `test_projects()` test, then `ci_proj` won't be available for the other tests to use. We should keep this test, but not have it effect global state, and have another way of setting up the project for the other tests (either a fixture or code within the test itself).